radio/src/targets/flysky/AFHDS2A.cpp

```diff
@@ -24,11 +24,12 @@
 
 extern int8_t s_editMode;
 
-inline uint32_t GetChipID(void) {
```

Comment:
Had to remove this to allow compilation. Adds some bytes, but anyway couldn't fit it on ELRSv2.

Reply:
How many bytes?